Source/WebCore/platform/libwpe/PlatformPasteboardLibWPE.cpp

```diff
@@ -72,7 +75,11 @@ String PlatformPasteboard::readString(size_t, const String& type) const
     if (!string.length)
         return String();
 
-    String returnValue({ string.data, string.length });
+    String returnValue;
+    if (string.length <= std::numeric_limits<size_t>::max())
+        returnValue = String({ string.data, static_cast<size_t>(string.length) });
+    else
+        returnValue = String({ string.data, std::numeric_limits<size_t>::max() });
```

Comment:
Same here, it looks like the goal is passing the smallest of `string.length` and `std::numeric_limits<sizt_t>::max()`.